src/gdrive/replace_open.sh

```diff
@@ -0,0 +1,5 @@
+#! /bin/bash
+
+# Basic script to replace opn(authorizeUrl, { wait: false }).then(cp => cp.unref()); with process.stdout.write(`Open this URL in your browser: ${authorizeUrl}`);
+
+sed -i 's/opn(authorizeUrl, { wait: false }).then(cp => cp.unref());/process.stderr.write(`Open this URL in your browser: ${authorizeUrl}\n`);/' node_modules/@google-cloud/local-auth/build/src/index.js
```

Comment:
Build-time workaround for trying to open user's browser from inside the container. Instead of triggering `xdg-open`, this prints a URL to `stderr`:

<img width="557" alt="Screenshot 2024-12-19 at 6 54 35 PM" src="https://github.com/user-attachments/assets/cd8fe7fc-9217-4933-9539-e954218b7cac" />